.github/workflows/release-process.yml

```diff
@@ -34,7 +34,7 @@ on:
 
 jobs:
   release_process:
-    if: github.repository == 'bytecodealliance/wasmtime'
+    if: "github.repository == 'bytecodealliance/wasmtime' || !github.event.schedule"
```

Comment:
What was this change for?

Reply:
Ah this was just to make this more easily testable in external repositories. Otherwise I couldn't, as written, test it anywhere but the wasmtime repo. The condition here allows explicit requests for this workflow via the github UI which in theory no one will ever do but me